lib/player.rb

```diff
@@ -0,0 +1,9 @@
+def get_player(default_name = 'Player 1')
+  player_name = gets.chomp
```

Comment:
- [ ] "gets" are not allowed in the lib folder. **Hint:** You can use a variable that 'gets' in your main and then use it in your method as a parameter, remember that `(name = 'Player 1')`  already works as a default, so if `name` is not provided, it will take 'Player 1', otherwise, it will take the name that was provided, so you don't need to do the comparison to know which one to use